spec/support/fixtures/canonical/sync/books.json

```diff
@@ -1,4 +1,28 @@
 [
+  {
+    "attributes": {
+      "title": "Cure Disease Potion Recipe",
```

Comment:
I moved this object to be the first one in the JSON array since it is used as the `item_in_json` for the specs, and for all the other specs the `item_in_json` is the first one in the fixture file.